twill11/hometask-5/4.8.sql

```diff
@@ -0,0 +1,30 @@
+DELETE FROM film_category WHERE film_id IN (
+ SELECT film_id FROM film 
+  WHERE title = 'Samurai Lion'
+)
```

Comment:
You can use **WITH** operator, https://postgrespro.ru/docs/postgresql/9.5/queries-with
```suggestion
with films_ids as (
    select film_id
    from film
    where title = 'Samurai Lion'
)

DELETE FROM film_category WHERE film_id IN (
 select film_id from films_ids
);
```
It helps you avoid duplicate requests.